### ISSUE TITLE
Model.train fails on object columns with different raw lengths

### ISSUE THREAD

**Issue by ang-zy** [sassoftware/python-dlpy#156]
# Setup
1. Run `dlpy.utils.create_object_detection_table()` on the VOC2012 data set
2. Split table using `dlpy.splitting.two_way_split()`
3. Instantiate a YOLOv2 model instance using `dlpy.applications.Tiny_YoloV2()`
4. Train the model witth the training table using `model.fit()`

# Error Log:
```
ERROR: The target variables _Object27_ and _Object26_ are mismatched. Ensure that all target variables share the same type (numeric, text, or nominal), the same data type, and the same format or format length.
ERROR: The action stopped due to errors.
```

# Preliminary findings:
When we checked the table in Data Explorer, we realized that the first 27 \_Object\_ columns have the same raw length and formatted length of 11. \_Object27\_, however, had a raw length of 9.
![image](https://user-images.githubusercontent.com/24840347/58787307-26d1ca00-861c-11e9-84ab-3b494fc10634.png)
This error occurred because none of the pictures has their 28th object label set to 'pottedplant', the longest and only label with 11 characters. This error should occur for any data set whose object label columns have different maximum lengths.

# Possible fix
Tracing the execution calls for model.fit(), we couldn't find any open-source code that could have caused this error message, so we surmised that the problem lies in the `deeplearn.dtrain` function.<br>
As a workaround, we re-imported the table with the \_Object\_ column types set to fixed length strings instead of variable length strings, and the `model.fit()` function ran.

**Comment by sasxianhu** [comment]
This is the design. You have to use the same meta data information for your columns. You can easily redefine the format using the format option in the table parms.
